src/unix.rs

```diff
@@ -130,6 +130,12 @@ impl Client {
             .parse()
             .map_err(|e| FromEnvErrorInner::CannotParse(format!("cannot parse `write` fd: {e}")))?;
 
+        // If either or both of these file descriptors are negative,
+        // it means the jobserver is disabled for this process.
```

Comment:
Will it be consistent with other code? I couldn't find any reference to [manual](gnu.org/software/make/manual/make.html) in other places.
Since it's "An implementation of the GNU make jobserver", isn't it assumed that all behavior is defined by this manual if not stated otherwise?

Reply:
Yeah, it is implied, though the other PR is doing something like this since @petrochenkov asked: https://github.com/rust-lang/jobserver-rs/pull/67/files#diff-b1a35a68f14e696205874893c07fd24fdb88882b47c23cc0e0c80a30c7d53759R594 (and generally I think it is a good idea to trace requirements).

Reply:
Link is added. Is it (right before `from_pipe` function) correct place/message?